### PR TITLE
documentation: many `teardown`s in one test

### PR DESCRIPTION
**Pull request by utanapishtim**
The documents `teardown`'s ability to accept multiple cleanup functions over the lifetime of a test.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -399,6 +399,23 @@ test('some test', async ({ ok, teardown }) => {
 })
 ```
 
+You can call `teardown` multiple times in a test and every function passed will be called after a test ends
```
Comment (davidmarkclements) [inline readme.md:402]: try to keep the same voice as the docs, the second person ("you") isn't used here

File: readme.md [hunk 2]
```diff
@@ -399,6 +399,23 @@ test('some test', async ({ ok, teardown }) => {
...
+
+```js
+import test from 'brittle'
+test('some test', async ({ ok, teardown }) => {
+ teardown(doSomeCleanUp)
+ const assert = test('some sub test')
+ setTime(() => {
```
Comment (davidmarkclements) [inline readme.md:409]: setTimeout

File: readme.md [hunk 3]
```diff
@@ -399,6 +399,23 @@ test('some test', async ({ ok, teardown }) => {
...
+   const resource = createSomeResourceThatNeedsCleaningUpLaterOn()
+   teardown(async () => { await resource.cleanup() })
+   assert.is(resource.methodThatReturnsABoolean(), true)
+ }, Math.random() * 1000)
```
Comment (davidmarkclements) [inline readme.md:413]: does randomness needs to be present to demonstrate the point?

**Review by davidmarkclements — CHANGES_REQUESTED** [review]
great! just a few nit picks

File: readme.md (hunk 2 above)
Comment (utanapishtim) [reply readme.md]: lol :facepalm:

File: readme.md (hunk 3 above)
Comment (utanapishtim) [reply readme.md]: not at all

**Review by davidmarkclements — APPROVED** [review]

**Pull request merged**